src/lib/rest/RestService.cpp

```diff
@@ -445,7 +487,7 @@ static bool compErrorDetect
 *
 * restService -
 */
-std::string restService(ConnectionInfo* ciP, RestService* serviceV)
+static std::string restService(ConnectionInfo* ciP, RestService* serviceV, const char* serviceVectorName)
```

Comment:
Not easy to find this kind of things in the github jungle ...
```
% find . -name "*.cpp" -exec grep "restService(" {} /dev/null
./src/lib/rest/RestService.cpp:static std::string restService(ConnectionInfo* ciP, RestService* serviceV, const char* serviceVectorName)
./src/lib/rest/RestService.cpp:    return restService(ciP, restBadVerbV, "BAD VERB");
./src/lib/rest/RestService.cpp:  if      ((ciP->verb == GET)     && (getServiceV     != NULL))    return restService(ciP, getServiceV,     "GET");
./src/lib/rest/RestService.cpp:  else if ((ciP->verb == POST)    && (postServiceV    != NULL))    return restService(ciP, postServiceV,    "POST");
./src/lib/rest/RestService.cpp:  else if ((ciP->verb == PUT)     && (putServiceV     != NULL))    return restService(ciP, putServiceV,     "PUT");
./src/lib/rest/RestService.cpp:  else if ((ciP->verb == PATCH)   && (patchServiceV   != NULL))    return restService(ciP, patchServiceV,   "PATCH");
./src/lib/rest/RestService.cpp:  else if ((ciP->verb == DELETE)  && (deleteServiceV  != NULL))    return restService(ciP, deleteServiceV,  "DELETE");
./src/lib/rest/RestService.cpp:  else if ((ciP->verb == OPTIONS) && (optionsServiceV != NULL))    return restService(ciP, optionsServiceV, "OPTIONS");
./src/lib/rest/RestService.cpp:  else                                                             return restService(ciP, restBadVerbV,    "BADVERB");
```

If I'm not mistaken, I used the "serviceVectorName" fior debugging purposes.
Need to make sure, but if so, it will be removed.

Reply:
The parameter is no longer used. Removed in ... next commit :)

Reply:
Fixed in 64983d136